### PR TITLE
feat: added styling for different charts IM-3364

### PR DESCRIPTION
**Pull request by ymchandra**
The following stying (Font label, font colour and font size) options have been added

- Axis label
- Axis title
- Value label

for the following charts

- Histogram
- Waterfall
- Distplot
- Boxplot
- Gauge

Jira: https://jira.qlikdev.com/browse/IM-3364

### CODE REVIEW

File: chart-modules/common/picasso/chart-builder/presets/dimension-measure-chart/dimension-measure-chart.js [hunk 1]
```diff
@@ -143,6 +143,7 @@ function addMeasureScale(chartBuilder, options, chartOrientation, measureAxisPro
 function addMeasureAxisComponent(chartBuilder, options, measureDirection, measureAxisProperties) {
   let measureAxisSettings = {
     scale: 'measure',
+    settings: options.settings || {},
```
Comment (T-Wizard) [inline chart-modules/common/picasso/chart-builder/presets/dimension-measure-chart/dimension-measure-chart.js:146]: I would like a more specific name. (not just settings)
(maybe something like axisTitleSettings or axisTitleStyle)

File: chart-modules/common/picasso/chart-builder/presets/dimension-measure-chart/dimension-measure-chart.js [hunk 2]
```diff
@@ -155,6 +156,7 @@ function addMeasureAxisComponent(chartBuilder, options, measureDirection, measur
 function addMeasureAxisTitleComponent(chartBuilder, options, measureDirection) {
   let measureTitleSettings = {
     text: options.measureTitleText,
+    style: options.style || {},
```
Comment (T-Wizard) [inline chart-modules/common/picasso/chart-builder/presets/dimension-measure-chart/dimension-measure-chart.js:159]: I would like a more specific name. (not just style)
(maybe something like axisTitleStyle)
(or maybe just have both this and new axisLabelSyle/setting with under style)

File: charts/boxplot/src/boxplot-view.js [hunk 3]
```diff
@@ -769,10 +770,12 @@ const BoxPlot = ChartView.extend('BoxPlot', {
         brushComponents: ['box-marker'],
       },
 
-      brushActions: this._dependentActions.gestures,
+      brushActions: this._dependentActions?.gestures,
 
       // ref-lines
       refLines: layout.refLine && layout.refLine.refLines,
+      ...getAxisTitleSettings(chartID, theme, layout),
+      ...getAxisLabelSettings(chartID, theme, layout),
```
Comment (T-Wizard) [inline charts/boxplot/src/boxplot-view.js:778]: I would like to not use ... here to reduce the risk of future property name collision.
(also apply to the other charts)

Comment (ymchandra) [reply charts/boxplot/src/boxplot-view.js]: Fixed.

File: chart-modules/common/picasso/chart-builder/presets/dimension-measure-chart/dimension-measure-chart.js (hunk 2 above)
Comment (ymchandra) [reply chart-modules/common/picasso/chart-builder/presets/dimension-measure-chart/dimension-measure-chart.js]: Changed the names to match the styles.

File: chart-modules/common/picasso/chart-builder/presets/dimension-measure-chart/dimension-measure-chart.js (hunk 1 above)
Comment (ymchandra) [reply chart-modules/common/picasso/chart-builder/presets/dimension-measure-chart/dimension-measure-chart.js]: Done.

**Review by T-Wizard — APPROVED** [review]

**Pull request merged**